apps/iglidur-designer/src/app/pages/model/model.component.html

```diff
@@ -39,10 +39,29 @@ <h2>Select Model</h2>
     />
     } }
 
-    <div id="model" [class.hidden]="screenshotUrl">
+    <!-- <div id="model" [class.hidden]="screenshotUrl"> -->
+    <div #rendererContainer [class.hidden]="screenshotUrl">
       @if (modelUrl && !screenshotUrl) {
       <stl-model-viewer [stlModels]="[modelUrl]"></stl-model-viewer>
       }
     </div>
   </div>
+  <div class="footer">
+    <p class="footer-text">Please rate this configurator</p>
+    <div class="rating">
+      <mat-icon
+        *ngFor="let star of stars; let i = index"
```

Comment:
prefer new control flow

In fact add it to your eslint

Reply:
I have added the rule to the eslint